qubesappmenus/__init__.py

```diff
@@ -249,6 +251,11 @@ def appmenus_create(self, vm, force=False, refresh_cache=True):
             anything_changed = True
             directory_changed = True
         appmenus = list(self.get_available_filenames(vm))
+        def make_desktop_name(vm, appmenu_basename):
+            return '.'.join((qubes_vm_desktop, vm.name, appmenu_basename))
+        def make_settings_name(vm):
+            return '.'.join((qubes_vm_desktop_settings, vm.name, 'desktop'))
+        join = os.path.join
```

Comment:
Please don't do this. Either use full `os.path.join` (preferable), or change import to `from os.path import join`. Otherwise it is very confusing why it's sometimes `join` and sometimes `os.path.join`.